basic-request.md

```diff
@@ -1,8 +1,8 @@
 ---
 
-copyright:
+Copyright:
```

Comment:
Any reason why this got capitalized. Not sure whether it makes a difference, but guidance is lower case: https://console.test.cloud.ibm.com/docs/developing/writing/markdown/tips.html#copyright

Reply:
I probably did it  by mistake with an Esc-c.  I'll change it now; tahnks.

Reply:
Fixed.  I checked; it's the only instance of this.